### PR TITLE
Multiple flows

### PR DESCRIPTION
**Pull request by angelika233**
## Summary
Added a new class named MultipleFlows which enables running multiple flows in given order

## Importance
User can define list of flows and run them in one flow. When one of them is not working main flow stops and following flows won't be executed

## Checklist

This PR:

- [x] adds new tests (if appropriate)
- [x] updates docstrings for any new functions or function arguments (if appropriate)
- [x] updates `CHANGELOG.md` with a summary of the changes

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -5,6 +5,8 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 ## [Unreleased]
 ### Added
+- Added `MultipleFlows` flow class which enables running multiple flows in a given order.
+### Added
```
Comment (Rafalz13) [inline CHANGELOG.md:9]: `###Added` section is duplicated

File: viadot/flows/multiple_flows.py [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+
+
+class MultipleFlows(Flow):
+    """Flow to run multiple flows in the given order.
+    Args:
+        flow_name(str): Name of a new flow.
+        flows_list(List[List]): List containing lists of flow names and project names - [["flow1_name" , "project_name"], ["flow2_name" , "project_name"]].
+            Flows have to be in the correct oreder. Defaults to [List[None]].
+    """
+
```
Comment (Rafalz13) [inline viadot/flows/multiple_flows.py:46]: Delete empty line

**Review by Rafalz13 — COMMENTED** [review]
CC: @m-paz

File: viadot/flows/multiple_flows.py (hunk 2 above)
Comment (angelika233) [reply viadot/flows/multiple_flows.py:46]: Black code formatter adds this line

**Pull request merged**